db/scripts/4_create_views.sql

```diff
@@ -52,7 +52,22 @@ WHERE
 ORDER BY l1.batch_index ASC
 ;
 
+
+CREATE OR REPLACE VIEW batchable_l2_only_tx_states
```

Comment:
Ahh, the tough part here is that we have tx batches and state root batches, so it's important to indicate that it is the state roots here that are batchable and not the txs, regardless of what data is returned.

Reply:
Just wanted to follow up on this. I'm thinking I'll just leave it since txs and state roots are stored in the same table for the L2 stuff (as compared to the separation in `l1_rollup_tx` and `l1_rollup_state_root`), and the name clearly indicates that the batchable thing is the state root. If you think of a better name that is still clear, let me know, and I'll be happy to change it!